.github/workflows/development-docker.yml

```diff
@@ -6,39 +6,104 @@ on:
       - fb-*
 
 jobs:
-  docker:
-    if: github.repository_owner == '0xERR0R'
+  vars:
     runs-on: ubuntu-latest
+    outputs:
+      tag: ${{ steps.get_all.outputs.tag }}
+      ghtag: ${{ steps.get_gh.outputs.tag }}
+    steps:
+      - name: Extract branch name as tag base
+        shell: bash
+        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+        id: extract_branch
+      
+      - name: Get GitHub tag
+        shell: bash
+        run: echo "##[set-output name=tag;]$(echo ghcr.io/${{github.repository}}:${{steps.extract_branch.outputs.branch}})"
+        id: get_gh
+
+      - name: Get all tags
+        shell: bash
+        run: |
+          RESPONSE="${{steps.get_gh.outputs.tag}}"
+          if [ "${{github.repository_owner}}" == '0xERR0R' ]
+          then
+            RESPONSE+="\nspx01/blocky:${{steps.extract_branch.outputs.branch}}"
+          fi
+          echo "##[set-output name=tag;]$(echo ${RESPONSE})"
+        id: get_all
+
+  docker-gitlab:
```

Comment:
gitlab?

Reply:
sorry should be github, had to many projects open 😐